test/components/views/rooms/SearchResultTile-test.tsx

```diff
@@ -77,11 +75,9 @@ describe("SearchResultTile", () => {
             />,
         );
 
-        const tiles = wrapper.find(EventTile);
+        const tiles = container.querySelectorAll<HTMLElement>(".mx_EventTile");
         expect(tiles.length).toEqual(2);
-        expect(tiles.at(0).prop("mxEvent").getId()).toBe("$1:server");
-        // @ts-ignore accessing private property
-        expect(tiles.at(0).prop("callEventGrouper").events.size).toBe(2);
```

Comment:
We seem to have lost test assertions here, due to not being able to introspect in RTL. Can we assert this in any other manner?

Reply:
Unfortunately no, the second event `$2:server` is not represented at all in the DOM